cylc/flow/network/server.py

```diff
@@ -15,7 +15,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Server for suite runtime API."""
 
-import getpass
+import getpass  # noqa: F401
```

Comment:
What's this F401 for?

Reply:
getpass is not used, but causes failures if it's removed.

Reply:
What failures occurred?  I just tried removing this line and didn't see any problems on running a flow.